src/hsmm_mvpy/visu.py

```diff
@@ -136,11 +171,12 @@ def plot_topo_timecourse(channels, estimated, channel_position, init, time_step=
         topo_size = .08 * timescale * magnify #8% of time scale
 
     #fix vmin/vmax across topos, while keeping symmetric
+    vals = channels[:] if not cond_plot else np.concatenate(channels)
     if vmax == None: #vmax = absolute max, unless no positive values
-        vmax = np.nanmax(np.abs(channels[:])) if np.nanmax(channels[:]) >= 0 else 0
-    if vmin == None: #vmin = absolute max, unless no negative values
-        vmin = -np.nanmax(np.abs(channels[:])) if np.nanmin(channels[:]) < 0 else 0
- 
+        vmax = np.nanmax(np.abs(vals)) 
+        vmin = -vmax if np.nanmin(vals) < 0 else 0
```

Comment:
This is exactly how it was, just different code. I personally really prefer to have a symmetric scale, as otherwise either positive or negative values are (strongly) over-represented.

Reply:
OK I guess most users would agree with that default